core/controllers/editor.py

```diff
@@ -659,16 +1078,55 @@ class ImageUploadHandler(EditorHandler):
     """Handles image uploads."""
 
     _decorator = None
+    URL_PATH_ARGS_SCHEMAS = {
+        'entity_type': {
+            'schema': {
+                'type': 'basestring'
+            }
+        },
+        'entity_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'POST': {
+            'image': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': 100 * 1024
```

Comment:
Why this value?

Reply:
Maybe remove it in case you don't see it in the existing handler?

Reply:
Removed.